js/events/index.d.ts

```diff
@@ -33,6 +33,11 @@ export interface NativeEventInfo<TComponent, TNativeEvent = Event> {
     readonly event?: DxEvent<TNativeEvent>;
 }
 
+/** @public */
+export interface NativeInputEventInfo<TComponent, TNativeEvent = Event> extends Omit<NativeEventInfo<TComponent, TNativeEvent>, 'event'> {
```

Comment:
I would make it more verbose for better readability:
1) Extract a helper type for replacing `target` prop
2) Do not use `omit` for `NativeEventInfo`

I believe that keeping plain fields structure in `EventInfo`, `NativeEventInfo` and this new `NativeInputEventInfo` makes these types easier to read.

Reply:
Smth like this:

```ts
/** @public */
export interface NativeInputEventInfo<TComponent, TNativeEvent extends Event = Event> {
    readonly component: TComponent;
    readonly element: DxElement;
    readonly model?: any;
    readonly event?: DxEvent<SpecifyTarget<TNativeEvent, HTMLInputElement>>;
}

type SpecifyTarget<TNativeEvent extends Event, TEventTarget extends EventTarget> = {
    target: TEventTarget;
} & Omit<TNativeEvent, 'target'>;
```